src/js/stores/AuthStore.js

```diff
@@ -18,12 +18,27 @@ import {
 import AppDispatcher from '../events/AppDispatcher';
 import AuthActions from '../events/AuthActions';
 import CookieUtils from '../utils/CookieUtils';
-import {Hooks} from 'PluginSDK';
+import PluginSDK, {Hooks} from 'PluginSDK';
 
 class AuthStore extends GetSetBaseStore {
   constructor() {
     super(...arguments);
 
+    PluginSDK.addStoreConfig({
+      store: this,
+      storeID: this.storeID,
```

Comment:
We need it because it is used here:
https://github.com/dcos/dcos-ui/blob/master/src/js/mixins/GetSetMixin.js#L43
https://github.com/dcos/dcos-ui/blob/master/src/js/plugin-bridge/AppReducer.js#L16

😢 Sad panda